zulipterminal/core.py

```diff
@@ -85,14 +85,15 @@ def draw_screen(self, *args: Any, **kwargs: Any) -> None:
 
     def show_help(self) -> None:
         help_view = HelpView(self)
+        dialog_height = min(50, help_view.height)
```

Comment:
50 is probably too compact. It displays all the way to 'Scroll to bottom', whereas the current version (with '-') displays to 'Send a message'. It takes <1/3 of my vertical screen space on a 12.5" screen.
![2019-02-12-123859_1522x767_scrot](https://user-images.githubusercontent.com/395821/52636119-9c762680-2ec3-11e9-9a23-387a98f74a32.png)

Reply:
@rht I just updated, using the height of the screen and taking the half.

Reply:
Yeah looks better / less cramped to me.

Reply:
Incidentally it is also 50% of screen on webapp, though not sure if it continues to stretch to be 50% on larger screen,